lib/_007/Parser/Actions.pm6

```diff
@@ -78,6 +82,24 @@ class _007::Parser::Actions {
     }
 
     method statement:expr ($/) {
+        if $<export> {
+            # For now, we're just enforcing that there's a `my` at the far left, according to the
```

Comment:
I am sure this is me being wrong, but I've look and looked at this.
1) A comment states that 'we're just enforcing that there's a "my" [...]' at line 86, but that is not true since...
2) more things a included: My, $, Prefix, Postfix and Infix. Ok. so that comment don't (if I am correct here) everything that happends but - and this is important (perhaps) - why is not func included in the same file (Actions.pm6). func is 'exportable' in the file Syntax.pm6

I guess it passes the test (line 13) in t.pm6? But... I am very confused and I don't know if this is me not understanding the code (the seperation of things) or if the structure is strange. To me infix, postfix (and so on) IS different types of functions, but apperently - for reasons beyond my capacity - it's seen as something else... Hmm. Is everything realy accurate here?

Anyways... The comment should be more inclusive? (Another solution would be erasing the My-part...)